src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+
+    }
```

Comment:
There are redundant empty lines after the method signature and before the closing brace of the method. According to the checklist, we should not add unnecessary empty lines in such places.